modules/http/src/main/resources/META-INF/mule-httpn.xsd

```diff
@@ -702,10 +702,10 @@
                 </xsd:restriction>
             </xsd:simpleType>
         </xsd:attribute>
-        <xsd:attribute name="sendBodyMode" default="AUTO">
+        <xsd:attribute name="sendBodyMode">
```

Comment:
Yes, though I'll cherry-pick the tests to make sure the 4.x extension is working as expected.